source/testing/index.md

```diff
@@ -5,39 +5,73 @@ This application would likely include models such as `user` and `post`. It would
+a failing test, develops the functionality to get the test passing, and then circles back to
+clean the code by doing things like removing duplication and poor naming
+(red->green->refactor).
+
+While unit tests are written against any isolated application logic, some specific
+examples for the scenario above might be:
+
+* A fullname attribute is computed which is the aggregate of its first and last.
+* The serialized properly converts the blog request payload into a blog post model object.
+* Blog dates are properly formatted.
+
+### Integration Tests
+
+Integration tests serve as a middle ground between acceptance tests, who only interact with the
+full system through user endpoints, and unit tests, who interact with specific code algorithms
+on a micro level. Integration tests verify interactions between various parts of the
+application, such as behavior between various UI controls.  They are valuable in ensuring
+data and actions are properly passed between various parts of the system, and provide
+confidence that parts of the system will work within the application under various scenarios.
+
+It is recommended that components be tested with integration tests because the component
+interacts with the system in the same way that it will within the context of the application,
+including being rendered from a template and receiving Ember's lifecycle hooks.
+
+Examples of integration tests from the above example might include:
 
-* A user has a role
-* A user has a username
-* A user has a fullname attribute which is the aggregate of its first and last
-  names with a space between
-* A post has a title
-* A post's title must be no longer than 50 characters
+* An author's full name and date are properly displayed in a blog post.
+* A user is prevented from typing more than 50 characters into post's title field.
+* Submitting a post without a title displays a red validation state on the field and gives
+the user text indicating that the title is required.
+* The blog post list scrolls to position a new post at the top of the viewport.
 
 ### Testing Frameworks
 
-[QUnit] is the default testing framework for this guide, but others are
-supported through third-party addons.
+[QUnit] is the default testing framework for this guide, but others are supported through
+third-party addons.
 
 ### How to Run Your Tests
 
-Run your tests with `ember test` on the command-line. You can re-run your tests on every file-change with `ember test --server`. These commands run your tests using [Testem] to make testing multiple browsers very easy. You can configure Testem using the `testem.json` file in your application root.
+Run your tests with `ember test` on the command-line. You can re-run your tests on every
+file-change with `ember test --server`. Another way to run the tests when you are running
+a local server (started by running `ember server`), is to navigate your browser to the address
```

Comment:
added a blurb about running tests from a running server, so devs don't have to run ember test and ember server simultaneously.